src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -1860,9 +1708,5 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
         });
     };
 
-    if (browser.chromecast) {
-        mediaManager = new cast.receiver.MediaManager(document.createElement('video'));
```

Comment:
Is `PlayOn` feature implemented by dedicated player?
If so, all the code related to `mediaManager` can also be removed, right?

Reply:
I don't think we ever even get to the Media Manager stuff, it's all hidden behind `if (mediaManager)` but that variable never gets assigned anything.